src/cart/crt_utils.c

```diff
@@ -56,6 +56,8 @@ crtu_drain_queue(crt_context_t ctx)
 	int	rc;
 	int	i;
 
+	crt_rank_abort_all(NULL);
```

Comment:
Looking at this commit history this is a test change to provoke the failure so shouldn't be landed?

Reply:
I hope this should test this race with double completion.

Reply:
this would test/trigger this race with double rpc completions. from test standpoint it is also not a bad thing to do to cancel all rpcs as we are shutting down the test/draining queue.